fe/fe-core/src/main/java/com/starrocks/catalog/CatalogUtils.java

```diff
@@ -235,4 +236,26 @@ public static int calBucketNumAccordingToBackends() {
         }
         return bucketNum;
     }
+
+    public static void maskCloudCredential(Map<String, String> properties) {
```

Comment:
You can create a new file into credential package, maybe named `CloudCredentialUtil.java`, then put this static method into here.😃